src/background/modules/AutoRemoteFollow.js

```diff
@@ -144,6 +144,16 @@ function getInteractionType(url) {
     return [null, null];
 }
 
+/**
+ * Handles changes to the URL of a tab.
+ * @returns {void}
+ */
+function onTabUpdate() {
+    browser.tabs.executeScript(null, {
```

Comment:
```suggestion
    browser.tabs.executeScript({
```

The first parameter is optional anyway, see https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/executeScript, so let's remove that.